src/Nodes.jl

```diff
@@ -38,8 +38,28 @@ function _getnode(cam, name::String, nodemap)
   return hNode
 end
 
+function _getnodemap(cam, nodemap)
```

Comment:
Should this not just be the non-mutating version of `_nodemap` and thus share its name without the bang? If so, this should go in `NodeMap.jl`.

Reply:
You're right. Updated